src/app.rs

```diff
@@ -54,7 +54,8 @@ impl App {
             .helper_threads(if config.env == ::Env::Production {3} else {1})
             .build();
         let diesel_db_config = r2d2::Config::builder()
-            .pool_size(if config.env == ::Env::Production {1} else {1})
+            .pool_size(if config.env == ::Env::Production {50} else {1})
```

Comment:
I thought prod had 120 connections available? I told it to not try to keep more than 5 idles at a time so it should never get bad on staging.